gnuboardProject/frontend/src/components/Portfolio/PortfolioCreate.tsx

```
@@ -0,0 +1,795 @@
+  };
+
+  // 섹션 내용 변경
+  const updateSectionContent = (type: string, value: string) => {
+    setSections(sections.map(section =>
+      section.type === type ? { ...section, content: value } : section
+    ));
+  };
+
+  // 핸들러 및 추가 함수도 필드명에 맞게 수정
+  const handleExperienceChange = (idx: number, field: string, value: string) => {
+    setExperiences(experiences.map((exp, i) => i === idx ? { ...exp, [field]: value } : exp));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)